astropy/tests/helper.py

```diff
@@ -3,15 +3,14 @@
 This module provides the tools used to internally run the astropy test suite
 from the installed astropy.  It makes use of the `pytest` testing framework.
 """
-
 import os
 import sys
 import types
 import pickle
 import warnings
 import functools
-
 import pytest
+from ..utils.decorators import deprecated
```

Comment:
As I'm nitpicking anyway, an open line before this statement (separating standard library from our own code).

Reply:
Remove this since it isn't used.